### DISCUSSION TITLE
tiny fire

### DISCUSSION THREAD

**Discussion by robotic-gamer** [CleverRaven/Cataclysm-DDA#48451] [Ideas]
-a long standing exploit is to drop a single piece of thread, set on fire then lure a zombie on it which does large amounts of damage
-a solution I can think of is to create a tier of fire below small fire that does little/no damage when stand on
-that way this trick is obsolete
-a semi related idea would be a cardboard box filled with paper/cardboard/birch bark that could be used like a Molotov cocktail but create a 1 by 1 flame instead of 3 by 3

edit: alternative to a cardboard box would be to tie it together with cordage

**Comment by actual-nh** [comment]
The first 3 of the above probably belong in an issue, BTW.

**Reply by robotic-gamer** [reply]
done, thanks for advice

**Comment by anothersimulacrum** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/issues/48466
